src/bootstrap/src/core/builder/cargo.rs

```diff
@@ -1215,22 +1215,6 @@ impl Builder<'_> {
             rustflags.arg("-Zinline-mir-preserve-debug");
         }
 
-<<<<<<< HEAD
-        if self.config.rustc_parallel
-            && matches!(mode, Mode::ToolRustc | Mode::Rustc | Mode::Codegen)
-        {
-            // keep in sync with `bootstrap/lib.rs:Build::rustc_features`
-            // `cfg` option for rustc, `features` option for cargo, for conditional compilation
-            rustflags.arg("--cfg=parallel_compiler");
-            rustdocflags.arg("--cfg=parallel_compiler");
-        }
-
-        if target.contains("ferrocenecoretest") {
-            rustflags.arg("-Zpanic-abort-tests");
-        }
```

Comment:
is this intentional

Reply:
nope, I should only have removed the `rustc_parallel` related code, I'll re add the `ferrocenecoretest` part.